From 052856923116439092565cc2953f61003c0eedc5 Mon Sep 17 00:00:00 2001
From: Kalev Lember <kalevlember@gmail.com>
Date: Fri, 22 Aug 2014 23:10:59 +0200
Subject: devhelp plugin: Support webkit2gtk-4.0

API changes between webkit2gtk-3.0 and webkit2gtk-4.0 were minor and we
can support both, depending on which one devhelp is built with.

https://bugzilla.gnome.org/show_bug.cgi?id=735251

diff --git a/configure.ac b/configure.ac
index 619f126..ec0a0fa 100644
--- a/configure.ac
+++ b/configure.ac
@@ -237,6 +237,9 @@ if test "x$devhelp_enabled" = "xyes"; then
 		*webkit2gtk-3.0*)
 			AC_DEFINE([HAVE_WEBKIT2], [1], [Defined if devhelp is built against webkitgtk2])
 			;;
+		*webkit2gtk-4.0*)
+			AC_DEFINE([HAVE_WEBKIT2], [1], [Defined if devhelp is built against webkitgtk2])
+			;;
 		esac
 fi
 AM_CONDITIONAL(HAVE_PLUGIN_DEVHELP, [test x$devhelp_enabled = xyes])
-- 
cgit v0.10.1

